src/Compilers/CSharp/Test/Syntax/Parsing/CollectionExpressionParsingTests.cs

```diff
@@ -5656,6 +5657,191 @@ public void CastVersusIndexAmbiguity30()
         EOF();
     }
 
+    [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/69508")]
+    public void CastVersusIndexAmbiguity31()
+    {
+        UsingStatement("var collection = (List<Cell>)[1, 2, 3, 4, 5];");
```

Comment:
>(List<Cell>)[1, 2, 3, 4, 5] [](http://example.com/codeflow?start=41&length=27)

Minor: perhaps simplify to `(A<B>)[1, 2]` in these two tests, to shorten the tests slightly.

Reply:
done.

Reply:
oh gotcha...